src/app/@theme-home/components/header/header.component.ts

```diff
@@ -18,21 +16,33 @@ export class HeaderComponent implements OnInit {
 
   user: any;
 
-  userMenu = [{ title: 'Profile' }, { title: 'Log out' }];
+  userMenu = [{ title: 'Perfil' }, { title: 'Sair' }];
 
   constructor(private sidebarService: NbSidebarService,
               private menuService: NbMenuService,
-              private userService: UserService,
               private analyticsService: AnalyticsService,
               private route: Router,
               private authService: AuthService,
             ) {
   }
 
   ngOnInit() {
-    this.userService.getUsers()
-      .subscribe((users: any) => this.user = users.nick);
-    this.toggleLogado();
+    this.menuService.onItemClick().subscribe(title => {
+      if (title.item.title === 'Sair') {
+        this.sair();
+      } else if (title.item.title === 'Perfil') {
+        this.route.navigate(['/home/conta']);
+      }
+    });
+    this.authService.onStateChange().subscribe(isLogged => { // Escuta mudanças de login do usuario
+      if (isLogged) {
+        this.authService.getResolvedUser().subscribe(user => {
```

Comment:
pra não ter que colocar um subscribe dentro do outro você pode fazer uma das duas coisas que vou sugerir a seguir:
```suggestion
         this.authService.onStateChange().pipe(
           switchMap(isLogged => {
             return this.authService.getResolvedUser();
           }),
         ).subscribe(....
```

ou até 

```suggestion
         this.authService.onStateChange().pipe(
           filter(isLogged => isLogged === true),
           map(() => this.authService.getResolvedUser())
         ).subscribe(....
```
Qualquer um resolve o problema de dar subscribe dentro de um subscribe, como o estado pode mudar, dois subscribes podem ser trigados em momentos diferentes podendo gerar um problema; O usuário logar com o fb e não alterar a barra.

você pode usar até outro operador como `flatMap`, há várias opções